src/store/store.jsx

```diff
@@ -0,0 +1,4 @@
+import { createStore } from "redux";
+import rootReducer from "src/store/reducers/rootReducer";
+
+export const store = createStore(rootReducer);
```

Comment:
Use this for devTool extension from browser.
```suggestion
export const store = createStore(
  rootReducer,
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
);
```

Reply:
@iud-wtag I am using a different approach 
1. Installed redux-devtools-extension with npm 
2. Import composeWithDevTools in store.jsx